lib/index.js

```diff
@@ -13,7 +13,9 @@ function createJSONRules() {
         '*': (x, y, acc) => acc * y,
         '%': (x, y, acc) => acc % y,
         includes: (arr, value) => {
-            if (Array.isArray(arr)) {
+            if (!arr) {
+                return breakFalse;
+            } else if (Array.isArray(arr)) {
```

Comment:
Logic should be "flipped" here. We should check for `if (Array.isArray(arr)) {` first (as it was previously doing), then check for any other edge cases before throwing a type error. e.g.:

````
if (Array.isArray(arr)) {
    const hasValue = arr.includes(value);

    return hasValue ? ['break;', true] : breakFalse;
}

if (!arr) {
    return breakFalse;
}

throw Error(`"includes" operator expected an array but received: ${arr}`);

````

as the majority of situations would be it being an array, we may as well check for that first!

Reply:
plus could you fixed the `recieved` typo